01-atoms/images/_background-image.scss

```diff
@@ -1,24 +1,42 @@
-// This css is paired with the js in `background-image.js` to allow us
-// to use fully responsive background images that are not possible with css alone.
-// Read that file for a little more detail.
-[data-responsive-background-image] {
-  background-size: cover;
-  background-position: center center;
+@mixin background-image {
+  & {
+    position: relative;
+    overflow: hidden;
+  }
+
+  &__image {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+  }
+
+  &__image img {
```

Comment:
This should probably include `picture` as well.

Reply:
Do you mean something like:
```
  &__image img,
  &__image picture {
```

Reply:
yep

Reply:
Done